gradoop-flink/src/main/java/org/gradoop/flink/model/impl/operators/subgraph/Subgraph.java

```diff
@@ -191,7 +191,8 @@ private LG vertexInducedSubgraph(LG superGraph) {
       .where(new TargetId<>()).equalTo(new Id<>())
       .with(new LeftSide<>());
 
-    return superGraph.getFactory().fromDataSets(filteredVertices, newEdges);
+    return superGraph.getFactory()
```

Comment:
Please add a description in javadoc that the graph head stays unchanged for the subgraph operator